wpimath/src/main/native/include/frc/kinematics/ChassisSpeeds.h

```diff
@@ -60,5 +60,24 @@ struct WPILIB_DLLEXPORT ChassisSpeeds {
     return {vx * robotAngle.Cos() + vy * robotAngle.Sin(),
             -vx * robotAngle.Sin() + vy * robotAngle.Cos(), omega};
   }
+
+  /**
+   * Converts a user provided field-relative ChassisSpeeds object into a robot-relative ChassisSpeeds
+   * object.
+   *
+   * @param fieldRelativeSpeeds The ChassisSpeeds object representing the speeds
+   * in the field frame of reference. Positive x is away from your alliance
+   * wall. Positive y is to your left when standing behind the alliance wall.
+   * @param robotAngle The angle of the robot as measured by a gyroscope. The
+   * robot's angle is considered to be zero when it is facing directly away from
+   * your alliance station wall. Remember that this should be CCW positive.
+   * @return ChassisSpeeds object representing the speeds in the robot's frame
+   * of reference.
```

Comment:
Please indent these so they're readable.

Reply:
I tried. The formatter failed when I did that. https://github.com/wpilibsuite/allwpilib/actions/runs/3300942177/jobs/5445933546#step:8:1

Reply:
That's because you made the lines too long, so they got wrapped by clang-format.